texastribune/donations-app
# Remove smd15 image; update og image to point to fmd image

#### What's this PR do?

Switches out old FB share image with new one for FMD.
#### Why are we doing this? How does it help us?

Helps us have our FB shares branded for the FMD.
#### How should this be manually tested?

Run the project locally with
`bundle exec middleman`
Inspect the homepage and other pages to see that the og:image element points to https://s3.amazonaws.com/static.texastribune.org/media/membership/FMD15-social_FB.png. Visit that url to see the FMD image. Because of the way FB works, it might not pull the image in for you when you test out the 'Share on Facebook' button. After deploying these changes, we'll have to ask FB to re-scrape support.texastribune.org and then check that it's correctly grabbing this image and displaying it with FB posts.
#### Are there performance implications? If adding JS can it be done async? Do images/CSS/JS have cache headers set?
#### What are the relevant tickets?

Off-sprint request.
#### Next steps?

Change it to a different image after the FMD is over.
#### Smells?
#### TODOs:
#### Has the relevant documentation/wiki been updated?
#### Technical debt note

Same
